x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimensions_editor_helpers.tsx

```diff
@@ -73,6 +75,9 @@ export const CalloutWarning = ({
           })}
           iconType="alert"
           color="warning"
+          css={css`
+            margin-bottom: ${euiTheme.size.base};
+          `}
```

Comment:
It looks like both this and the next callout are using the same exact inline CSS. Is it worth removing those `css` props and instead apply these styles as a single style by targeting the `.lnsIndexPatternDimensionEditor__warning` class selector in `dimension_editor.scss`? Doing so would avoid repeated code and avoid implementing competing style methodologies (old Sass way and new Emotion way), at least until we decide to tackle a transition to Emotion in a more holistic fashion.

Reply:
Got it!